### PR TITLE
Docs: Adds v0.13.0 Release notes

### PR DESCRIPTION
**Pull request by dgasmith**
## Description
Changelog for the v0.13.0 release.

## Changelog description
N/A

## Status

- [x] Code base linted
- [x] Ready to go

### CODE REVIEW

File: docs/qcfractal/source/changelog.rst [hunk 1]
```diff
@@ -15,6 +15,37 @@ Changelog
...
+
+Enhancements
+++++++++++++
+- (:pr:`507`) Automatically adds collection molecules in chunks if more than the current limit needs to be submitted.
+- (:pr:`515`) Conda environments now correspond to docker images in all deployed cases.
+- (:pr:`524`) The ``delete_collection`` function was added to ``qcportal.FractalClient``.
+- (:pr:`530`) Adds the ability to specify cores per rank for node-parallel tasks in QCEngine.
+- (:pr:`507`) Adds a formatting and lint check to CI during pull requests.
+- (:pr:`535`) Allows dftd3 to be computed for all stoichiometries rather than just defaults.
+- (:pr:`542`) Important: TaskRecord.base_result is now an ObjectId (int or str), and no more a ``DBRef``. So, code that uses ``my_task.base_result.id`` should change to simply use ``my_task.base_result``.
```
Comment (mattwelborn) [inline docs/qcfractal/source/changelog.rst]: Can we mark this as a Breaking Change?

Comment (dgasmith) [reply docs/qcfractal/source/changelog.rst]: This seemed minor and internal enough that I was going to gloss over it as this should not effect user space. I don't think we have marked anything as so yet. Do you have a recommended way of doing this?

Comment (mattwelborn) [reply docs/qcfractal/source/changelog.rst]: Does it break our fractal examples for running and checking on calculations?

Comment (dgasmith) [reply docs/qcfractal/source/changelog.rst]: It does not.

Comment (mattwelborn) [reply docs/qcfractal/source/changelog.rst]: Then I'm happy with the changelog as is.

**Pull request merged**